recipes/elephas/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "elephas" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5cb28cb23827b45d96f5af59379cd7f739ee9e5785e9b61fe3a22440d4fea528
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3
+
+  run:
+    - cython
+    - flask
+    - keras
```

According to the ``setup.py``, ``hyperas`` is a dependency os this package
Should we package ``hyperas`` first?

---

I created this PR https://github.com/conda-forge/staged-recipes/pull/9159 and also add you as a maintainer of that recipe.
After we package that project we can continue with this one

---

@marcelotrevisani  great, I was just about to push Hyperas myself, thanks!

---

@marcelotrevisani and thanks for adding me as a maintainer!

---

Now you can add ``hyperas`` and it should be fine! :)
```suggestion
    - keras
    - hyperas
```